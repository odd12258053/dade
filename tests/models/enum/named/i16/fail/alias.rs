use dade::model;
#[model]
enum TestModel {
    Value {
        #[field(alias = 1)]
        value: i16
    },
}
fn main() {}
