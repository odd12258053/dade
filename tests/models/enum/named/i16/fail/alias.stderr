error: custom attribute panicked
 --> tests/models/enum/named/i16/fail/alias.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: Error("expected string literal")
